go/pkg/db/schema.sql

```diff
@@ -399,6 +415,8 @@ CREATE INDEX `idx_keys_on_workspace_id` ON `keys` (`workspace_id`);
 CREATE INDEX `owner_id_idx` ON `keys` (`owner_id`);
 CREATE INDEX `identity_id_idx` ON `keys` (`identity_id`);
 CREATE INDEX `deleted_at_idx` ON `keys` (`deleted_at_m`);
+CREATE INDEX `workspace_id_idx` ON `credits` (`workspace_id`);
+CREATE INDEX `refill_lookup_idx` ON `credits` (`refill_day`,`refill_amount`,`refilled_at`);
```

Comment:
Isn't cardinality too high on this one? Too many refill days 1-31 and amounts can be anything int32 🤔